.gitignore

```diff
@@ -17,5 +17,10 @@ hypnotoad.pid
 perl5
 perl5/*
 
+#Database Container Storage
+data
+data/*
+~data/.keep
```

Comment:
Should there be a leading `~` here? I'm not aware of tilde meaning anything to gitignore

Reply:
Yes, you are right. 
This was rather meant to keep the file and the directory so it will be distributed to new installations.